app/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.tsx

```diff
@@ -340,12 +340,15 @@ const DirectoryTable: React.FunctionComponent<DirectoryTableProps> = ({
   }
 
   return (
-    <div
-      className="directory-table pb-2 mt-4"
-      style={{ width: "auto", maxWidth: "100%" }}
-    >
+    <>
       {experiments.length ? (
-        <table className="table" data-testid="DirectoryTable">
+        <Table
+          data-testid="DirectoryTable"
+          size="sm"
+          responsive
+          hover
+          style={{ width: "100%" }}
```

Comment:
Does this need double `{}`?

Reply:
yup, because - To style an element with the inline style attribute `style={{width:"100%"}}`, the value must be a JavaScript object. In JSX, JavaScript expressions are written inside curly braces, and since JavaScript objects also use curly braces, the styling in the example above is written inside two sets of curly braces `{{}}.`